.github/workflows/build-natives.yaml

```diff
@@ -75,6 +75,32 @@ jobs:
           for /f "tokens=3 delims=<> skip=1" %%v in ('findstr "<version>" ..\main\pom.xml') do set WILDFLY_OPENSSL_NATIVES_VERSION=%%v
           echo Running tests with WildFly OpenSSL Natives %WILDFLY_OPENSSL_NATIVES_VERSION%
           mvn -B verify --file pom.xml -Dorg.wildfly.openssl.path="C:\vcpkg\installed\x64-windows\tools\openssl" -Dversion.org.wildfly.openssl.natives=%WILDFLY_OPENSSL_NATIVES_VERSION%
+      - if: matrix.os == 'ubuntu-latest'
+        name: Install latest version of OpenSSL in ubuntu
+        run: |          
+          echo "Installed version of OpenSSL on ubuntu is "
+          openssl version
```

Comment:
Should this line be moved to the end of this section to ensure the version is correct after installing and building?

Reply:
Oh maybe this line is meant to check the existing version first?

Reply:
Ah right, I was using this to check the existing version first but it can be removed now

Reply:
@fjuma  ubuntu does not package 3.x versions, they just patch CVEs when needed to older versions IIUC. So I had to install manually and I cca followed this tutorial https://www.howtoforge.com/tutorial/how-to-install-openssl-from-source-on-linux/

Reply:
This uses 3.0.7 since that is the one that got used in windows but we should configure windows version as well, I created https://issues.redhat.com/browse/SSLNTV-36 for now